tox.ini

```diff
@@ -31,13 +29,6 @@ deps =
 commands = pre-commit run --all-files
 
 
-[testenv:licenses]
```

Comment:
Why drop this?

Reply:
It kept failing, so I removed it for the moment to get further, but will definitely return it back once I figure how to make it work :)